src/test/java/edu/ksu/canvas/attendance/submitter/CanvasAssignmentAssistantUTest.java

```diff
@@ -150,7 +150,7 @@ public void editAssignmentInCanvasCanvasAssignmentNotFound() throws IOException,
     }
 
     @Test
-    public void deleteAssignmentInCanvasHappyPath() throws IOException, AttendanceAssignmentException {
+    public void deleteAssignmentInCanvasHappyPath() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00112)